AppiumTests/AppiumTests.swift

```diff
@@ -2,17 +2,21 @@
 //  AppiumTests.swift
 //  AppiumTests
 //
-//  Created by Sharath G Pai on 16.08.17.
+//  Created by Mahmoud Adam on 10/12/17.
 //  Copyright © 2017 Mozilla. All rights reserved.
```

Comment:
As you update the creator, probably should change the Copyright to Cliqz as well